src/e2e-test/features/servicenowsink/RunTime.feature

```diff
@@ -0,0 +1,332 @@
+@SNSink
+@Smoke
+@Regression
+Feature: ServiceNow Sink - Run time scenarios
+
+  @TS-SN-RNTM-SINK-01 @BQ_SOURCE_TEST_RECEIVING_SLIP_LINE
+  Scenario: Verify user should be able to preview the pipeline when plugin is configured for Insert operation
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Data Pipeline - Batch"
+    And Select plugin: "BigQuery" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Fill Reference Name
+    And Configure BigQuery source plugin for Dataset and Table
+    And Validate "BigQuery" plugin properties
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "ServiceNow" from the plugins list
+    And Connect plugins: "BigQuery" and "ServiceNow" to establish connection
+    And Navigate to the properties page of plugin: "ServiceNow"
+    And Fill Reference Name
+    And fill Credentials section for pipeline user
+    And Enter input plugin property: "tableName" with value: "receiving_slip_line"
+    And Select radio button plugin property: "operation" with value: "INSERT"
+    And Validate "ServiceNow" plugin properties
+    And Close the Plugin Properties page
+    And Preview and run the pipeline
+    Then Verify the preview of pipeline is "success"
+    And Click on the Preview Data link on the Sink plugin node: "ServiceNow"
+    And Verify sink plugin's Preview Data for Input Records table and the Input Schema matches the Output Schema of Source plugin
+
+  @TS-SN-RNTM-SINK-02 @BQ_SOURCE_TEST_RECEIVING_SLIP_LINE
+  Scenario: Verify user should be able to deploy and run the pipeline plugin is configured for Insert operation
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Data Pipeline - Batch"
+    And Select plugin: "BigQuery" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Fill Reference Name
+    And Configure BigQuery source plugin for Dataset and Table
+    And Validate "BigQuery" plugin properties
+    And Capture the generated Output Schema
+    And Close the Plugin Properties page
+    And Select Sink plugin: "ServiceNow" from the plugins list
+    And Connect plugins: "BigQuery" and "ServiceNow" to establish connection
+    And Navigate to the properties page of plugin: "ServiceNow"
+    And Fill Reference Name
+    And fill Credentials section for pipeline user
+    And Enter input plugin property: "tableName" with value: "receiving_slip_line"
+    And Select radio button plugin property: "operation" with value: "INSERT"
+    And Validate "ServiceNow" plugin properties
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    And Verify the pipeline status is "Succeeded"
+    Then Validate OUT record count is equal to IN record count
```

Comment:
for test cases where we run a deployed pipeline, can we also verify that the data written to the sink is correct (not just record count)?

Reply:
Sure, I have created a new Step which fetch response from both the source and sink applications and compare those responses.(commit - bc007e5ad85fb475bbb1152c81a8209c78268a06)